bench/tiered/push_pull.c

```diff
@@ -214,11 +218,15 @@ recover_validate(const char *home, uint32_t num_records, uint64_t file_size, int
     testutil_wiredtiger_open(opts, home, buf, NULL, &conn, true, true);
     testutil_check(conn->open_session(conn, NULL, NULL, &session));
 
-    // srand((unsigned long)getpid() + num_records);
-    srand((uint32_t)getpid() + num_records);
+    /* Seed the random number generator */
+    v = (uint32_t)getpid() + num_records + (2 * counter);
+    w = (uint32_t)(2 * flush) + num_records;
+    z = (2 * counter) + num_records;
```

Comment:
See the other comment - we probably want to use a single 64 bit number that includes some/all of these inputs: getpid(), num_records, counter.  But getting a random seed for this use case is probably not so important.

Reply:
Fixed